plugin/cmd/kube-scheduler/app/configurator.go

```diff
@@ -194,3 +210,87 @@ func (sc schedulerConfigurator) Create() (*scheduler.Config, error) {
 
 	return sc.CreateFromConfig(*policy)
 }
+
+func (sc *schedulerConfigurator) SetupPolicyConfigMapEventHandlers(client clientset.Interface, informerFactory informers.SharedInformerFactory) {
+	// selector targets only the scheduler's policy ConfigMap.
+	selector := cache.NewListWatchFromClient(client.CoreV1().RESTClient(), "configmaps", sc.policyConfigMapNamespace, fields.OneTermEqualSelector(api.ObjectNameField, string(sc.policyConfigMap)))
+
+	sharedIndexInformer := informerFactory.InformerFor(&clientv1.ConfigMap{}, func(client clientset.Interface, resyncPeriod time.Duration) cache.SharedIndexInformer {
+		sharedIndexInformer := cache.NewSharedIndexInformer(
```

Comment:
> It is the case, but I added it in a previous PR to the defaults.go file for componentconfig. I am not sure if it's the best way. I can change it if it's not the right approach.

I think it's probably confusing to have the two defaults be different... I'd suggest changing the one I pointed to to kube-system namespace.